apps/nextjs-website/react-components/components/Accordion/AccordionItem.tsx

```diff
@@ -39,15 +39,23 @@ export const AccordionItem: React.FC<
   };
 
   useLayoutEffect(() => {
-    if (itemID && window.location.hash === `#${itemID}`) {
-      const targetItem = document.getElementById(itemID);
-
-      if (targetItem) {
-        setExpanded(true);
-        targetItem.scrollIntoView({ behavior: 'smooth', block: 'center' });
+    const handleHashChange = () => {
+      if (itemID && window.location.hash === `#${itemID}`) {
+        const targetItem = document.getElementById(itemID);
+        if (targetItem) {
+          setExpanded(true);
+          targetItem.scrollIntoView({ behavior: 'smooth', block: 'start' });
```

Comment:
```suggestion
          targetItem.scrollIntoView({ behavior: 'smooth', block: 'center' });
```
Setting block: 'start' will hide the Accordion under the Header